config/bootstrap.php

```diff
@@ -207,11 +207,3 @@
  * Plugin::load('Migrations'); //Loads a single plugin named Migrations
```

Comment:
Shouldn't this comment also be removed?

Reply:
whole comments or just `Plugin::load('Migrations...` ?